.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
-        id: generate-release-notes
+        uses: actions/github-script@v4.0.2
+        with:
```

Comment:
- Ensure proper error handling and logging mechanisms are in place to handle unexpected scenarios.